app/code/core/Mage/Tax/Model/Calculation/Rate/Title.php

```diff
@@ -48,6 +49,10 @@ protected function _construct()
         $this->_init('tax/calculation_rate_title');
     }
 
+    /**
+     * @param $rateId
```

Comment:
int

Reply:
Comes from ...

```
        $fieldName = $this->getIdFieldName();
        if ($fieldName) {
            return $this->_getData($fieldName);
        } else {
            return $this->_getData('id');
        }
```

... should be `int`, but i guess it is `string` .... however .... changed